src/templates/index.html

```diff
@@ -42,7 +42,17 @@
     </div>
     <footer class="footer">
       <div class="container-fluid">
-        <p class="text-muted" style="font-size:12px;"><b>Disclaimer:</b>&nbsp;The package information is provided AS-IS with no representations whatsoever with respect to accuracy and/or completeness. We reserve the right to change the information at any time and without notice. To obtain the most current package information, please visit the official linux distribution websites.</p>
+        <p class="text-muted" style="font-size:12px;">
+          <b>Disclaimer:</b>
+          &nbsp;The package information is provided AS-IS with no representations whatsoever with respect to accuracy and/or completeness. We reserve the right to change the information at any time and without notice. To obtain the most current package information, please visit the official linux distribution websites.
+          <br>
+          <b>Contribute:</b>
+          Source code for software discovery tool is available <a href="https://github.com/openmainframeproject/software-discovery-tool/">here</a>.  
+          You can find the project description <a href="https://www.openmainframeproject.org/projects/software-discovery-tool">here</a>.
```

Comment:
Let's drop this line and instead include the link down below, where noted :arrow_down: